etl/src/main/scala/etl/RecipeParsing.scala

```diff
@@ -79,8 +80,10 @@ object RecipeParsing {
 
   def guessSteps(body: Seq[Element]): Seq[String] = {
     val candidates: Seq[Element] = body.filter(NumberedParagraph.matches)
-    candidates.map(_.text).filter(_.nonEmpty)
+    val stepsFound = candidates.map(_.text).filter(_.nonEmpty)
+    val pattern = new scala.util.matching.Regex("""(^\d+.?\s?)(.*+)""", "number", "step")
```

Comment:
Yep all true

> Scala regex class delegates to the java.util.regex package of the Java Platform.

> An instance of Regex represents a compiled regular expression pattern. Since compilation is expensive, frequently used Regex es should be constructed once, outside of loops and perhaps in a companion object.